### PR TITLE
Switch to alpine:3.3 and other fixes

### PR DESCRIPTION
**Pull request by AFDudley**
Please apply https://github.com/eris-ltd/eris-cli/pull/671 before testing. Thank you!

### CODE REVIEW

**Comment by zramsay** [comment]
@pietv are we good on this? let's merge :) will help lighten images => good for ARM

File: docker/ipfs/Dockerfile [hunk 1]
```diff
@@ -58,7 +58,7 @@ USER $USER
 WORKDIR $ERIS
 
 # configure
-RUN mkdir --parents $IPFS_PATH
+RUN mkdir -p $IPFS_PATH
```
Comment (zramsay) [inline docker/ipfs/Dockerfile]: we should use long flags for anything hard-coded. it's not documented anywhere but has been requested in Slack at some point.

**Comment by pietv** [comment]
LGTM :+1:

File: docker/ipfs/Dockerfile (hunk 1 above)
Comment (AFDudley) [inline docker/ipfs/Dockerfile]: That long comment doesn't work in alpine/busybox, that's why I changed it.

**Comment by AFDudley** [comment]
LGTM.

File: docker/ipfs/Dockerfile (hunk 1 above)
Comment (zramsay) [inline docker/ipfs/Dockerfile]: good to know!

**Pull request merged**